cypress/integration/prospects/note.js

```diff
@@ -0,0 +1,39 @@
+describe('Create a prospect note', () => {
+	it('Should be able to create a prospect note', () => {
```

Comment:
Indentation should only have 2 spaces in Typescript

Reply:
In my version of the code, the indentation is correct, why did github take it like that?

![Screenshot (647)](https://github.com/Wellcode-PB/SalesWell/assets/124595318/027fc359-e0db-4749-861a-1c88a1c50a66)

Reply:
you need to make sure that you use spaces everywhere